_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Vorzeigeprojekte
+  blog: Blog
+  community: Gemeinschaft
+  assets: Assets
+  download: Download
+  docs: Docs
+  contribute: Mitwirken
+  donate: Spenden
+  language: Sprache
+footer:
+  download: Download
+  documentation: Dokumentation
+  features: Features
+  web_editor: Web Editor
+  release_archive: Release Archiv
+  source_code: Quellcode
+  project: Projekt
+  blog: Neuigkeiten
+  code_of_conduct: Verhaltenskodex
+  governance: Verwaltung
+  teams: Teams
+  priorities: Prioritäten
+  communities: Gemeinschaften
+  resources: Ressourcen
+  asset_library: Asset-Bibliothek
+  press_kit: Presse-Kit
+  showcase: Vorzeigeprojekte
+  education: Bildung
+  foundation: Stiftung
+  about: Über uns
+  donate: Spenden
+  license: Lizenz
+  privacy_policy: Datenschutzerklärung
+  contact_us: Kontakt
+  text: "Juan Linietsky, Ariel Manzur und <a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>Beitragende</a>. Gehostet von der <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot Foundation</a>. Website Quellcode <a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>auf  GitHub</a>."
+sponsors:
+  title: "Godot wird gesponsert durch:"
+  text: Wenn Sie an einem Unternehmensponsoring interessiert sein, schreiben Sie bitte an <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>.<br>Spenden als Einzelpersonen können durch den <a href='https://fund.godotengine.org/'>Godot Engine Entwicklungsfonds</a> getätigt werden.
+  cta: Spenden
+home:
+  h1: Deine kostenlose, Open&#8209;Source<br>Game Engine.
+  subtitle: Entwickle 2D- &amp; 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+  whats_new: Neuigkeiten
+  download_latest: Downloade die neueste Version
+  looking_for: 'Auf der Suche nach <a href="/download/windows/" class="set-os-download-url" data-version="3" title="Download der für Langzeit unterstützten Version von Godot 3">Godot 3</a>, unseren <a href="/download/archive">experimentellen Veröffentlichungen</a>, oder <a href="/download/archive">früheren Versionen</a>?'
+  latest_news: Aktuelle Nachrichten
+  more_news: Mehr Nachrichten
+  cards:
+    h2: Eine andere Art Spiele zu machen
```

This one doesn't sound great in german IMO. First, "machen" sounds... a bit simple, and overall this doesn't really sound that positive. Again, neither a marketing or comms person, just wanted to note.

---

At the top of my head how about:
> Eine erfrischende Art Spiele zu entwickeln

instead?

---

```suggestion
    h2: Eine neue Art, Spiele zu entwickeln
```
I tend to overuse punctuation marks, but that sentence needs one for grammar reasons